### DISCUSSION TITLE
TI trains, no error, yet nothing happens when i use it

### DISCUSSION THREAD

**Discussion by mildfever** [bmaltais/kohya_ss#1617] [Q&A]
So i tried training a TI embedding in Kohya, something simple: a person with 64 img + .txt captions.
It trains apparently ok, produces the .safetensor file, but when i try to apply it on a SD WebUI using the trigger word, it produces no results, as in the persons semblance does no render. 

Any idea of what i'm doing wrong??

This the .json content after it finishes training:

{
  "adaptive_noise_scale": 0,
  "additional_parameters": "",
  "bucket_no_upscale": true,
  "bucket_reso_steps": 64,
  "cache_latents": true,
  "cache_latents_to_disk": false,
  "caption_dropout_every_n_epochs": 0.0,
  "caption_dropout_rate": 0,
  "caption_extension": ".txt",
  "clip_skip": "1",
  "color_aug": false,
  "enable_bucket": true,
  "epoch": 1,
  "flip_aug": false,
  "full_fp16": false,
  "gradient_accumulation_steps": 1.0,
  "gradient_checkpointing": false,
  "init_word": "*",
  "keep_tokens": "0",
  "learning_rate": 0.005,
  "logging_dir": "",
  "lr_scheduler": "constant",
  "lr_warmup": 0,
  "max_data_loader_n_workers": "0",
  "max_resolution": "512,512",
  "max_timestep": 1000,
  "max_token_length": "75",
  "max_train_epochs": "",
  "max_train_steps": "",
  "mem_eff_attn": false,
  "min_snr_gamma": 0,
  "min_timestep": 0,
  "mixed_precision": "fp16",
  "model_list": "runwayml/stable-diffusion-v1-5",
  "multires_noise_discount": 0,
  "multires_noise_iterations": 0,
  "no_token_padding": false,
  "noise_offset": 0,
  "noise_offset_type": "Original",
  "num_cpu_threads_per_process": 2,
  "num_vectors_per_token": 1,
  "optimizer": "AdamW8bit",
  "optimizer_args": "",
  "output_dir": "/content/drive/My Drive/AI_PICS/model",
  "output_name": "MA_Natasha_P",
  "persistent_data_loader_workers": false,
  "pretrained_model_name_or_path": "runwayml/stable-diffusion-v1-5",
  "prior_loss_weight": 1.0,
  "random_crop": false,
  "reg_data_dir": "",
  "resume": "",
  "sample_every_n_epochs": 50,
  "sample_every_n_steps": 25,
  "sample_prompts": "photo of MA_Natasha_P eating a hotdog",
  "sample_sampler": "euler_a",
  "save_every_n_epochs": 1,
  "save_every_n_steps": 0,
  "save_last_n_steps": 0,
  "save_last_n_steps_state": 0,
  "save_model_as": "safetensors",
  "save_precision": "fp16",
  "save_state": false,
  "scale_v_pred_loss_like_noise_pred": false,
  "sdxl": false,
  "sdxl_no_half_vae": true,
  "seed": "",
  "shuffle_caption": false,
  "stop_text_encoder_training_pct": 0,
  "template": "caption",
  "token_string": "MA_Natasha_P",
  "train_batch_size": 3,
  "train_data_dir": "/content/drive/My Drive/images",
  "use_wandb": false,
  "v2": false,
  "v_parameterization": false,
  "vae": "",
  "vae_batch_size": 0,
  "wandb_api_key": "",
  "weights": "",
  "xformers": true
}
